cmake/BLTInstallableMacros.cmake

```diff
@@ -428,37 +428,80 @@ endmacro(blt_add_target_compile_flags)
 
 
 ##------------------------------------------------------------------------------
-## blt_convert_to_system_includes(TARGET <target>)
+## blt_convert_to_system_includes(TARGETS [<target>...]
```

Comment:
I would prefer not to break this interface. If you want to add `TARGETS` I would be okay with that but not remove `TARGET`

Reply:
I left `TARGET` as an option, but marked it as deprecated so that people have a chance to change before a breaking release.